packages/openstef-beam/src/openstef_beam/backtesting/backtest_forecaster/openstef4_backtest_forecaster.py

```diff
@@ -40,6 +42,10 @@ class OpenSTEF4BacktestForecaster(BaseModel, BacktestForecasterMixin):
         default=False,
         description="When True, saves intermediate input data for debugging",
     )
+    contributions: bool = Field(
+        default=False,
+        description="When True, saves intermediate input data for explainability",
```

Comment:
I feel like this description is a bit vague.
```suggestion
        description="When True, saves intermediate prediction contributions of input data for explainability",
```